src/config.rs

```diff
@@ -622,6 +622,8 @@ create_config! {
     merge_derives: bool, true, "Merge multiple `#[derive(...)]` into a single one";
     binop_separator: SeparatorPlace, SeparatorPlace::Front,
         "Where to put a binary operator when a binary expression goes multiline.";
+    match_arm_forces_newline: bool, false,
```

Comment:
I would call this `match_arm_force_block`. The description should mention blocks rather than newlines too.